get.sh

```diff
@@ -227,7 +227,7 @@ getBinaryOpenjdk()
 		info_url="https://api.adoptopenjdk.net/v3/assets/feature_releases/${JDK_VERSION}/${release_type}?architecture=${arch}&heap_size=${heap_size}&image_type=jdk&jvm_impl=${JDK_IMPL}&os=${os}&project=jdk&vendor=adoptopenjdk"
 
 		if [ "$JDK_VERSION" != "8" ] || [ "$JDK_IMPL" != "hotspot" ]; then
```

Comment:
When I read this if JDK_IMPL not hotspot, I presume this means its setting for openj9, double-checking with @sophia-guo and @llxia, is this change valid for this case?

Reply:
This is specific for openj9, jdk8 testimage. Looks like adoptium api doesn't support it. So may just leave as is.